### PR TITLE
Addtocart

### PR DESCRIPTION
**Pull request by leonardodeutsch**
- added add items to cart
- will fix conflicts in css

### CODE REVIEW

File: client/src/components/Overview/AddToCart.jsx [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
+
+  handleAddToCart(e) {
+    console.log({sku_id: Number(this.state.selectedSKU_id), quantity: this.state.selectedQuantity});
+    axios.post(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-nyc/cart`, { headers: {Authorization: API_KEY}, params: {sku_id: this.state.selectedSKU_id} })
+      .then((response) => {
+        console.log('successfully added to cart', response);
+      })
+      .catch((error) => {
+        throw error;
+      })
```
Comment (ks10825n) [inline client/src/components/Overview/AddToCart.jsx:42]: I wonder if you also need to make a get request to get the shopping cart state just for data persistence. But, I'm imagining the cart in the API has probably lots of stuff with no way to tie it to the current user without cookies. 

Approving since it seems like it's out of scope and not in the business requirements!

**Review by ks10825n — APPROVED** [review]
Looks good! Just left a thought about a GET request to get the shopping cart. Don't think you need to do anything

File: client/src/components/Overview/AddToCart.jsx (hunk 1 above)
Comment (leonardodeutsch) [reply client/src/components/Overview/AddToCart.jsx:42]: Yes I tried making a get request to see what was in it, it is just empty. Like you said, because we don't have an authentication system or anything there is no way to link the api database to a specific user
